source/base/partitioner.cc

```diff
@@ -474,7 +413,7 @@ namespace Utilities
             }
           ghost_indices_subset_data = ghost_indices_subset;
         }
-    }
+    } // namespace MPI
```

Comment:
I don't think that this is the end of the namespace.

Reply:
You are right - that was inserted automatically by `clang-format` when I had a bug in the code (one closing brace too much), but apparently not removed again.